config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -352,7 +353,8 @@ presubmits:
         - --ginkgo-parallel=1
         - --build=quick
         - --cluster=
-        - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=false
+        - --env=KUBE_FEATURE_GATES=AllAlpha=true,InTreePluginGCEUnregister=false,DisableCloudProviders=false,DisableKubeletCloudCredentialProviders=false
```

Comment:
don't we want these to be `DisableCloudProvider=true`  and `DisableKubeletCloudCredentialProviders=true` ?

Reply:
we don't have to decide now. Let's dig out from under where we are and then decide what more we need to test (and what variations are needed)